packages/shared/src/features/onboarding/shared/PricingPlansV2.tsx

```diff
@@ -0,0 +1,233 @@
+import React, { useMemo } from 'react';
+import classNames from 'classnames';
+import type { FunnelStepPricingPlan } from '../types/steps/pricing';
+import { RadioItem } from '../../../components/fields/RadioItem';
+import {
+  Typography,
+  TypographyColor,
+  TypographyTag,
+  TypographyType,
+} from '../../../components/typography/Typography';
+import { usePaymentContext } from '../../../contexts/payment/context';
+import type { ProductPricingPreview } from '../../../graphql/paddle';
+import ConditionalWrapper from '../../../components/ConditionalWrapper';
+import { PricingPlanVariation } from './PricingPlan';
+
+interface PricingPlansV2Props {
+  onChange?: (plan: string) => void;
+  items: Array<Pick<FunnelStepPricingPlan, 'priceId' | 'label' | 'badge'>>;
+  value?: string;
+}
+
+interface PricingPlanProps
+  extends Pick<
+    FunnelStepPricingPlan,
+    'priceId' | 'label' | 'badge' | 'oldPrice' | 'variation'
+  > {
+  isActive?: boolean;
+  onChange?: (planId: string) => void;
+  price?: ProductPricingPreview;
+}
+
+const PricingPlan = ({
+  isActive = false,
+  onChange,
+  ...plan
+}: PricingPlanProps) => {
+  const {
+    priceId,
+    variation,
+    label,
+    badge,
+    price: productOption,
+    oldPrice,
+  } = plan;
+  const { price } = productOption ?? {};
+  const isBestValue = variation === PricingPlanVariation.BEST_VALUE;
+
+  return (
+    <ConditionalWrapper
+      condition={isBestValue}
+      wrapper={(children) => (
+        <div
+          className={classNames(
+            'rounded-16 p-1',
+            isActive
+              ? 'bg-gradient-funnel-best-price text-white'
+              : 'bg-border-subtlest-secondary text-text-tertiary',
+          )}
+        >
+          <Typography
+            bold
+            type={TypographyType.Footnote}
+            className="p-1 pt-0 text-center"
+          >
+            BEST VALUE
+          </Typography>
+          {children}
+        </div>
+      )}
+    >
+      <RadioItem
+        className={{
+          wrapper: classNames(
+            'flex flex-col gap-2 rounded-16 bg-white px-2 py-3',
```

Comment:
The bg-white here wouldn't do anything due to overwrite in the isActive statement below.